### PR TITLE
Use a SSH secret for Katello RPM pushing

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy [hunk 1]
```diff
@@ -12,6 +12,11 @@ void push_rpms_direct(repo_source, repo_dest, overwrite = true, merge = false) {
     }
 }
 
+void push_rpms_katello(version) {
+    sshagent(['katello-fedorapeople']) {
+        sh "ssh katelloproject@fedorapeople.org \"cd /project/katello/bin && sh rsync-repos-from-koji ${version}\""
+    }
+
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy]:
```suggestion
}
```

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy]: Just saw it too.

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy [hunk 2]
```diff
@@ -12,6 +12,12 @@ void push_rpms_direct(repo_source, repo_dest, overwrite = true, merge = false) {
     }
 }
 
+void push_rpms_katello(version) {
+    sshagent(['katello-fedorapeople']) {
+        sh "ssh katelloproject@fedorapeople.org \"cd /project/katello/bin && sh rsync-repos-from-koji ${version}\""
```
Comment (evgeni) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy]:
```suggestion
        sh "ssh katelloproject@fedorapeople.org 'cd /project/katello/bin && sh rsync-repos-from-koji ${version}'"
```

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy]: Why is this needed? I'm not quite sure where the variable interpolation happens.

Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy]: `version` is a groovy var here, groovy replaces vars in double quotes, so what the shell sees doesn't have any vars and does not need to be in double quotes.

Comment (evgeni) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy]: also, it's not *needed*, I just thought not having to escape the " is nice

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy]: Updated

**Pull request merged**